cegs_portal/search/tsv_templates/v1/reg_effects.py

```diff
@@ -1,23 +1,126 @@
-def reg_effects(get_data):
-    reg_effects_page = get_data
+from cegs_portal.search.helpers.options import is_bed6
+from cegs_portal.search.templatetags.custom_helpers import format_float, if_strand
+
+
+def tested_element(source, target):
+    return f"{source.chrom_name}:{source.location.lower}-{source.location.upper}:{if_strand(source.strand)}:{target.name if target else 'N/A'}"
```

Comment:
The places where you call this function also tests if target is None (`tested_element(source, target) if target else "N/A"`). Which is correct - having the full name or having just "N/A"? My preference is for the full name, but with a small change:

```suggestion
    return f"{source.chrom_name}:{source.location.lower}-{source.location.upper}:{if_strand(source.strand)}{f':{target.name}' if target else ''}"
```

Reply:
per Alex, leave the colon.